e2eshark/pytorch/models/whisper-base/model.py

```diff
@@ -0,0 +1,45 @@
+import sys, argparse
+import torch
+import torch.nn as nn
+import torch_mlir
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+# import from e2eshark/tools to allow running in current dir, for run through
+# run.pl, commutils is symbolically linked to allow any rundir to work
+sys.path.insert(0, "../../../tools/stubs")
+from commonutils import E2ESHARK_CHECK_DEF
+
+# Create an instance of it for this test
+E2ESHARK_CHECK = dict(E2ESHARK_CHECK_DEF)
+
+
+test_modelname = "openai/whisper-base"
+tokenizer = AutoTokenizer.from_pretrained(test_modelname)
+model = AutoModelForCausalLM.from_pretrained(
+    test_modelname,
+    num_labels=2,
+    output_attentions=False,
+    output_hidden_states=False,
+    torchscript=True,
+)
+model.to("cpu")
+model.eval()
+prompt = "What is nature of our existence?"
+encoding = tokenizer(prompt, return_tensors="pt")
+E2ESHARK_CHECK["input"] = encoding["input_ids"].cpu()
+E2ESHARK_CHECK["output"] = model(E2ESHARK_CHECK["input"])
+model_response = model.generate(
+    E2ESHARK_CHECK["input"],
+    do_sample=True,
+    top_k=50,
+    max_length=100,
+    top_p=0.95,
+    temperature=1.0,
+)
```

Comment:
Are any of these parameters useful to tune in these tests? Would a developer want a way to set them across all models in the repo (e.g. with a CLI flag or environment variable)?

Reply:
It's just to give out some example output to the user. It isn't used for anything on developer/testing end.